test/test_quantization.py

```diff
@@ -610,6 +615,37 @@ def checkQuantized(model):
         checkQuantized(model)
 
 
+@unittest.skipIf(
+    not torch.fbgemm_is_cpu_supported(),
```

Comment:
Nit: I would expect `fbgemm_is_cpu_supported` to be private, or live in a different namespace as `torch`, but I suppose this is an artifact on how we bind operators in Python?

Reply:
this is a native function:  https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L1429
why should it be private?

Reply:
it's an implementation detail - polluting the global namespace is bad. We should just bind it directly to _C with pybind and expose through torch.quantization.backends.bla (or just reuse the available engines you have in another diff).

At the very least, we should prepend an underscore to it. But it all deserves a separate diff probably.

Reply:
Btw, I fixed it in a separate PR: #26840